aqua/QtPass/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               qt5 1.0
+PortGroup               qmake5 1.0
+
+github.setup            IJHack QtPass 1.4.0 v
+revision                0
+categories              aqua security
+license                 GPL-3+
+maintainers             nomaintainer
+platforms               any
```

Comment:
Why `platforms any`?

Reply:
QtPass also works on other BSDs and Linux.
But I just realised that in this case the destroot function should be inside an if Darwin conditional.

Reply:
But Qt5 is not supported on _multiple_ Darwin platforms. Also, `platforms any`, AFAIK, implies identical binaries, which obviously is not the case.

Reply:
So, on the contrary, it should have `{darwin > 10}` or whatever is the minimum threshold. Then, `freebsd linux` can be also added.